Sources/SentryCrash/Recording/SentryCrash.h

```diff
@@ -59,6 +59,11 @@ static NSString *const SENTRYCRASH_REPORT_ATTACHMENTS_ITEM = @"attachments";
 /** Init SentryCrash instance with custom base path. */
 - (id)initWithBasePath:(NSString *)basePath;
 
+/** Optional, custom cache directory. Use when default one can not be accessed, e.g. in security
+ * environment. Default: @c nil .
+ */
```

Comment:
`m`: That comment isn't entirely accurate. The class isn't public, so users don't interact with it. It should always be set to the customCacheDirecotyPath from the options anyways, and there is not default.